### PR TITLE
Nov 15 notes

### PR DESCRIPTION
**Pull request by hstone102**

### CODE REVIEW

File: Notes/2018-11-15.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>November 15, 2018}</title>
```
Comment (bmizepatterson) [inline Notes/2018-11-15.html]: Looks like you have a typo here
```suggestion
    <title>November 15, 2018</title>
```

Comment (hstone102) [reply Notes/2018-11-15.html]: Ok I fixed it.  Thank you for catching that.

**Comment by devinepk** [comment]
+1

**Comment by bmizepatterson** [comment]
👍

**Pull request merged**